gym/lib/gym/error_handler.rb

```diff
@@ -175,6 +179,71 @@ def print_xcode_path_instructions
         UI.message ""
       end
 
+      def print_environment_information
+        if Gym.config[:export_method].to_s == "development"
+          UI.message("")
+          UI.error("Your `export_method` in gym is defined as `:development`")
+          UI.error("which can't be used for `ipa` files for beta or App Store distribution")
+          UI.error("Please make sure to define a correct `export_method` when calling")
```

Comment:
Totally pedantic, but the line wrapping on these strings is a bit off and makes them a bit weird to read - the "which" on the second line could probably fit at the end of the first line. 🤓